source/drivers/driver-compatibility-reference.txt

```diff
@@ -1077,27 +1183,69 @@ MongoDB Compatibility
      - MongoDB 2.4
      - MongoDB 2.6
      - MongoDB 3.0
+     - MongoDB 3.2
+     - MongoDb 3.4
+
+   * - 1.0
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+
+   * - 0.7
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+
+   * - 0.6
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+     -
+
+   * - 0.5
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     -
+     -
 
    * - 0.4
      - |checkmark|
      - |checkmark|
      - |checkmark|
+     -
+     -
 
    * - 0.3
      - |checkmark|
      - |checkmark|
      -
+     -
+     -
 
    * - 0.2
      - |checkmark|
      - |checkmark|
      -
+     -
+     -
 
    * - 0.1
      - |checkmark|
      -
      -
+     -
+     -
 
+- Motor 1.0 wraps PyMongo 3.3+
+- Motor 0.7 wraps PyMongo 2.9.4+
```

Comment:
Motor 0.7 wraps PyMongo >= 2.9.4  and < 3.0. That is, it does not work with PyMongo 3.x.